test/bin/lockfile-version.t/run.t

```diff
@@ -0,0 +1,24 @@
+We have a simple project with a single package that we will use to generate
+a lockfile for our current version.
+
+  $ gen-minimal-repo
+  $ opam-monorepo lock > /dev/null
+  $ opam show --no-lint --raw -fx-opam-monorepo-version ./lockfile-version.opam.locked
+  "0.3"
+
+This is our current version.
+At the moment we are not backward compatible so if we downgrade this version number,
+pull will refuse the lockfile and suggest it is regenerated with the current plugin:
+
+  $ sed -i "s/x-opam-monorepo-version: \"0.3\"/x-opam-monorepo-version: \"0.2\"/" lockfile-version.opam.locked
+  $ opam-monorepo pull > /dev/null
```

Comment:
Should you run `pull` in the happy path as well to make sure it doesn't result in an error?

Reply:
Unfortunately pull will require internet access, which it won't have in the CI.